[{"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329183.3536389, "message": "is it like that in epio.ini?", "group_id": 2873, "id": 1177506}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329217.023916, "message": "or my paste of it here at least", "group_id": 2873, "id": 1177514}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329217.846916, "message": "app ID?", "group_id": 2873, "id": 1177515}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329244.6367149, "message": "'stores'", "group_id": 2873, "id": 1177522}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306328992.779026, "message": "ehmm, right. and it seems to. typo, never mind...", "group_id": 2873, "id": 1177449}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329840.5915041, "message": "yeah, that's probably the only way for now", "group_id": 2873, "id": 1177703}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329128.7823291, "message": "hmm, getting an error after all: [cron] Invalid time specifation 'sync_something '//example.com/example.xml some_param = 0 2 * * **' for 'app/manage.py sync_something http'", "group_id": 2873, "id": 1177487}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329655.93326, "message": "no, the syntax in ini files is pretty particular", "group_id": 2873, "id": 1177638}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329677.0736189, "message": "ah, of course, colon is also a valid delimiter", "group_id": 2873, "id": 1177645}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329231.6131301, "message": "/name", "group_id": 2873, "id": 1177519}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329617.340574, "message": "can i just single quote the url as in regular cron job commands?", "group_id": 2873, "id": 1177623}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306328807.109566, "message": "I have this management task that imports an XML file and parses it. It would need to run daily. having a bit of trouble with the [cron] format though. i would expect this line to work:", "group_id": 2873, "id": 1177406}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329178.3732989, "message": "in your pasted error the last two asterisks have no space between them", "group_id": 2873, "id": 1177503}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306328959.5669351, "message": "app/manage.py sync_something http://example.com/example.xml some_param = 0 2 * * *", "group_id": 2873, "id": 1177437}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329197.7108331, "message": "no just in the error", "group_id": 2873, "id": 1177510}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329513.126663, "message": "@tijs you have two equals signs in the line, it's splitting on the first", "group_id": 2873, "id": 1177594}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329533.874028, "message": ".... http = //media ....", "group_id": 2873, "id": 1177598}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329591.453769, "message": "that's actually a colon not an equal sign as in http://media...", "group_id": 2873, "id": 1177617}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329698.7525749, "message": "hmm, so how do i pass a URL as a parameter to the management command then?", "group_id": 2873, "id": 1177653}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329738.529815, "message": "not sure you can with the current syntax; I'm just checking that ConfigParser has no option for this", "group_id": 2873, "id": 1177670}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306329863.3761511, "message": "we're going to move the configuration file to JSON at some point soon, we think, but that'll be a few weeks at least", "group_id": 2873, "id": 1177708}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306330190.4093649, "message": "no problem", "group_id": 2873, "id": 1177782}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329765.234925, "message": "ok thanks. guess i'll have to add the urls to setting, that would work too i guess.", "group_id": 2873, "id": 1177678}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306329932.5758159, "message": "ok thanks for the help. not that much of an issue for this project but good to find the root cause :)", "group_id": 2873, "id": 1177726}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306397186.996232, "message": "hey @andrewgodwin one more thing... it can't find 'app/manage.py'. from the 'djangoproj/manage.py' example in the docs i was assuming i had to use the app/ dir to point to my django project since that would be the default path ep.io uses for projects right? i'm not using the checkout option... any hints?", "group_id": 2873, "id": 1189922}, {"user_id": 8740, "stars": [], "topic_id": 36736, "date_created": 1306415099.8464911, "message": "@tijs No, exclude the app dir if your django project is directly in the root of your checkout. That example assumes your django project is inside a folder in your repository.", "group_id": 2873, "id": 1191555}, {"user_id": 29686, "stars": [], "topic_id": 36736, "date_created": 1306494915.2312231, "message": "@andrewgodwin ok great, thanks!", "group_id": 2873, "id": 1203725}]